src/js/step.js

```diff
@@ -102,15 +99,17 @@ export class Step extends Evented {
       'destroy',
       'hide',
       'isOpen',
-      'render',
       'scrollTo',
+      'setupElements',
       'show'
     ]);
     this.setOptions(options);
     this.bindAdvance = bindAdvance.bind(this);
     this.bindButtonEvents = bindButtonEvents.bind(this);
     this.bindCancelLink = bindCancelLink.bind(this);
-    this.setupPopper = setupPopper.bind(this);
+    this.setupTooltipElem = setupTooltipElem.bind(this);
```

Comment:
Agree with steps that are agnostic to the lib as we can

Reply:
👍